content/docs/concepts/flops/overview.md

```diff
@@ -0,0 +1,158 @@
+
+  ### Apache's 'Data Suite'
+  FLOps works with real data on learner devices.
+  Several technologies provided by [Apache](https://www.apache.org/) are employed to efficiently receive, store, and retrieve such data.
+  - [Arrow](https://arrow.apache.org/docs/format/Columnar.html) columnar format for working with data in memory.
+  - [Parquet](https://parquet.apache.org/) columnar file format for working with data stored on disk.
+  - [Arrow Flight](https://arrow.apache.org/docs/format/Flight.html) a gRPC-based framework to transport Apache formatted data over the network.
+
+  ### [Buildah](https://buildah.io/)
+  Buildah helps FLOps to build containerized FL images inside running containers.
+  Therefore distributing the computational load of image building across devices.
+
+  ### [Anaconda Suite](https://docs.anaconda.com/)
+  FLOps uses the Anaconda Suite to resolve dependencies to provide equal ML training across devices.
+  To be more specific:
+   - [Miniconda](https://docs.anaconda.com/miniconda/) for lightweight (base) images
+   - [Mamba](https://www.anaconda.com/blog/a-faster-conda-for-a-growing-community) for fast dependency resolving
+
+{{< /details >}}
+
+{{< details "Convenient Installation & native CLI" >}}
+  FLOps can be easily set up by cloning its [repository](https://github.com/oakestra/addon-FLOps) and running the docker-compose file on the same machine as the Oakestra Root Orchestrator.
+  See the [FLOps Manuals](/docs/manuals/flops) for concrete instructions.
+
+  The [Oakestra CLI](/docs/getting-started/deploy-app/with-the-cli/#the-oak-cli) supports a [set of commands](/docs/manuals/cli/features/flops-addon/#oak-addon-flops) to work with FLOps.
+{{< /details >}}
+
+{{< details "Designed for Change" >}}
+  FLOps aims to be easily modifiable and extendable by developers and researchers.
+  FLOps employs the following techniques to strive for high code quality and flexibility:
+  - Use state-of-the-art libraries and frameworks
+  - Extendable CLI
+  - Ready-made extendable multi-platform images and services to automate development and evaluation workflows.
+  - Additional base images with optional development flags to speed up the build and execution times.
+  - Enforcement of proper styling and typing via formatters and linters, including CI.
+{{< /details >}}
+
+As a result, individuals with different levels of expertise in FL, automation, DevOps, containerization, and orchestration can benefit from these techniques and perform FL.
+
+{{< callout context="note" title="Naming Explained" icon="outline/info-circle" >}}
+  **FLOps** stands for *Federated Learning Operations*.
+  It is inspired by the field of *MLOps* that enhances ML with DevOps practices.
+
+  We do not claim ownership of this term or name;
+  on the contrary, we want to spread awareness of the benefits of applying DevOps and other techniques to FL/ML to improve the onboarding, usage, and development of the entire field.
+{{< /callout >}}
```

Comment:
No need to add this. If we get credits for "FLOps" name, I will happily take it

Reply:
I have removed the lower paragraph for now. Let me know if I should remove the entire callout instead.